rts/lra/test/tck/src/test/resources/logging.properties

```diff
@@ -15,9 +15,9 @@
 #
 
 handlers = java.util.logging.ConsoleHandler
-java.util.logging.ConsoleHandler.level = INFO
+java.util.logging.ConsoleHandler.level=INFO
```

Comment:
In another place in this PR you explicitly added spaces around the = yet here you have removed them. Any particular reason or is it just you noticing something that doesn't look quite right?